app/helpers/samples_helper.rb

```diff
@@ -143,7 +143,9 @@ def display_attribute_value(value, attribute, options = {})
       when Seek::Samples::BaseType::CV
         seek_cv_attribute_display(value, attribute)
```

Comment:
have you tested with the single cv case? I wonder whether this may also now need the `.html_safe`

Reply:
tested, no problem without `.html_safe`